src/i18n/translations/pl.json

```
@@ -0,0 +1,73 @@
+{
+  "dash": {
+    "header": {
+      "your_account_heading": "Twoje konto",
+      "account": "Konto",
+      "settings": "Ustawienia",
+      "log_out": "Wyloguj się"
+    },
+    "sidebar": {
+      "logo_by": "od",
+      "tournament": {
+        "catname": "turniej",
+        "overview": "Panel główny",
+        "ladder": "Drabinka turniejowa",
+        "standings": "Rankingi"
+      },
+      "tournament_data": {
+        "catname": "informacje o turnieju",
+        "competing_teams": "Drużyny",
+        "debate_motions": "Tezy debat",
+        "staff_judges_bias": "Organizatorzy, sędziowie i powiązania",
```

I know the initiative to reflect bias came from me, but a direct translation is way too long and looks comically off in the menu itself. Perhaps this would be more compact while keeping with the idea.
```suggestion
        "staff_judges_bias": "Personel i powiązania",
```

---

Yeah, that's a good idea.